src/deps.ts

```diff
@@ -1,4 +1,4 @@
 export { readAll } from "https://deno.land/std@0.99.0/io/util.ts";
 export { BaseSlackAPIClient } from "https://deno.land/x/deno_slack_api@0.0.2/base-client.ts";
 export { createManifest } from "https://deno.land/x/deno_slack_builder@0.0.7/manifest.ts";
-export { parse } from "https://deno.land/std@0.99.0/path/mod.ts";
```

Comment:
this wasn't used and I needed `flags`' `parse` function

Reply:
I think in general Deno recommends not to import mod.ts and import the related file immediately, something like this: https://github.com/slackapi/deno-slack-builder/pull/24/files

Reply:
Humm but there's only a mod.ts in the module here https://github.com/denoland/deno_std/tree/main/flags
I'm not sue what else I'd import